src/pages/login/forms/FormLogin.tsx

```diff
@@ -1,29 +1,25 @@
+import { Form } from "./Form";
 import { detect } from "detect-browser";
-import { useHistory } from "react-router-dom";
-
 import { useContext } from "preact/hooks";
-
+import { useHistory } from "react-router-dom";
 import { OperationsContext } from "../../../context/revoltjs/RevoltClient";
 
-import { Form } from "./Form";
-
 export function FormLogin() {
     const { login } = useContext(OperationsContext);
     const history = useHistory();
 
     return (
         <Form
             page="login"
-            callback={async (data) => {
+            callback={async data => {
                 const browser = detect();
                 let device_name;
                 if (browser) {
-                    const { name, os } = browser;
-                    if (window.isNative) {
-                        device_name = `Revolt Desktop on ${os}`;
-                    } else {
-                        device_name = `${name} on ${os}`;
-                    }
```

Comment:
Why is the native detection being removed?

Reply:
That appears to be a typo

Reply:
I'll add a commit adding it back